# debian_alsaconf_noclear.patch by Thomas Hood
#
# Don't clear the screen. (Let the user see error messages.)
Index: alsa-utils-1.0.18/alsaconf/alsaconf.in
===================================================================
--- alsa-utils-1.0.18.orig/alsaconf/alsaconf.in	2008-11-02 20:59:25.000000000 +0100
+++ alsa-utils-1.0.18/alsaconf/alsaconf.in	2008-11-02 20:59:26.000000000 +0100
@@ -483,10 +483,19 @@
   $DIALOG --msgbox "$msg" 17 60 || acex 0
 }
 
+clear() {
+    echo
+}
+
 # Exit function
 acex() {
   cleanup
-  clear
+  if [ "$1" = 0 ] ; then
+      clear
+  else
+      # Don't clear error messages
+      echo
+  fi
   exit $1
 }
 
